src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -57,7 +57,7 @@ protected override void SerializeCore(XmlElement nodeElement, SaveContext contex
 
             var helper = new XmlElementHelper(outEl);
             helper.SetAttribute("value", SerializeValue());
-            nodeElement.AppendChild(outEl);
+            nodeElement.ReplaceChild(outEl,nodeElement.FirstChild);
```

Comment:
Why we need to replace first child with outE1 instead of adding outE1 as first child?